.travis.yml

```diff
@@ -0,0 +1,39 @@
+notifications:
+  slack:
+    secure: cuB9waMrZiRnJjJLq68NTSg1nB+ypbCTy4cphu0VjfuDYhmQc4MgnkvmzLntsE9X9Oj8kSP51rRjD+IJJ8qT1GTMi/5XAqKNzncONEdx5T0Tict0xxAaY8OcumOV68tisQ1jEeAVA65zcvPoyWOVv+ha4MdHLx8+J85h/zOFMRALf2tJ8tUnC2iHidWCYsQS8vQEkKp1eDWJRdcJY+OrmeV5tlgWXP56b6pRdnhoWwOkwtRols6yoPw1j6uNewb3N2uvJIIw5g7ji209nXFM+/HFGLLqNlWUpqickbaZEG/m5VqeuxeaZTl6jlW2w2+B5zyMPKR8NHeSQ8+c6Si08LbgQDGkLh1xHNQOTxqTEOHUOsLgBm7Y+NX1QDfIfclvY+dhWhUiNHVT39+om/I/8MvpuDLLtujijJIq+mluJdjuAkWWXCVqYGMll0SalvEQf43Os6yxZnewJ0QjlajUY8txgPR9yyVLQduddiMymSIwAAKE3gLlhTpl59W3kRcKcd0tTUtnqH7KAg6GBnFqkqnUPv9Fmws+eOvyThg36gX7+RWbo/mwQZCH265qY6cf02pbsU9Hoo5HTXd7nWi4G/NL3vKR3yySrOCzfzgwY2vozwrWY09pKphNtRhtLZkXEKdvDyFt9i6/nh6muiaZEQ22dxxK/1iBKTeR+IW5p48=
```

Comment:
This is the wrong secure string.  Remove the notifications section and I will add it in a separate pull request.